docs/profile_dependencies.md

```diff
@@ -0,0 +1,157 @@
+---
+title: About InSpec Profile Dependencies
+---
+
+# Profile Dependencies
+
+An InSpec profile can bring in the controls and custom resources from another InSpec profile. Additionally, when inheriting the controls of another profile, a profile can skip or even modify those included controls.
```

Comment:
You use two different descriptions - "bring in" and "inherit". You should pick which way you want to phrase it. Or, if you want to switch, you should:

`
An InSpec profile can inherit - or "bring in" - the controls and custom resources of another InSpec profile. When inheriting the controls of another profile, the profile can skip, and, or modify those inherited controls.
`
This keeps the terminology the same throughout the thought stream.